var/spack/repos/builtin/packages/templight/package.py

```diff
@@ -83,13 +83,6 @@ class Templight(CMakePackage):
 
     # Clang debug builds can be _huge_ (20+ GB), make sure you know what you
     # are doing before switching to them
```

Comment:
this comment isn't relevant anymore... I think we can have a single explanation of the rationale in `build_systems/cmake.py`.